common/lib/xmodule/xmodule/modulestore/tests/factories.py

```diff
@@ -231,6 +233,7 @@ def _create(cls, target_class, **kwargs):
         parent_location = kwargs.pop('parent_location', None)
         data = kwargs.pop('data', None)
         category = kwargs.pop('category', None)
+        name = kwargs.pop('name', None)
```

Comment:
I think you should not modify item factory instead you should create a static tab in a course in your tests like this
```
course.tabs.append(
                    CourseTab.load('static_tab', name=name, url_slug='url_slug')
                )
                store.update_item(course, user_id)
```